### ISSUE TITLE
Add an interface for adding / removing custom fields from Simplemap

### ISSUE THREAD

**Issue by arnaudbroes** [semperfiwebdesign/simplemap#34]
Add an interface for adding / removing custom fields from Simplemap

**Comment by wpsmort** [comment]
This is a very popular request, especially among users who try to use SimpleMap as a directory.  A good example of this is the work we did for a client where we added custom fields for text and images.  See screenshots from that project below:

![profile-new](https://cloud.githubusercontent.com/assets/17548525/16094266/6f279290-330d-11e6-97cb-39beb12c8a20.jpg)
![serps-new](https://cloud.githubusercontent.com/assets/17548525/16094265/6f270082-330d-11e6-9126-f231a3e0fe4b.jpg)

**Comment by michaeltorbert** [comment]
@wpsmort Do we have this code somewhere?

**Comment by michaeltorbert** [comment]
@wpsmort Was this a custom version of SM or a plugin we wrote?

**Comment by wpsmort** [comment]
@michaeltorbert Yes, this was a custom version that Peter created.  It was an early version of his SimpleMap v2.5 beta that he'd worked on.

**Comment by wpsmort** [comment]
We have a backup of the client site that we did this development work for.  The backup is in the usual location for customer backups.

**Comment by wpsmort** [comment]
Requested again here - http://support.simplemap-plugin.com/discussions/questions/1519-custom-fields
